### PR TITLE
add aircraft data

### PR DESCRIPTION
**Pull request by kornrumpfl**
Different airplane models to be displayed on the select box

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -44,6 +45,9 @@ export default function Main({ onHandleSubmit }) {
     setArrivalRunway(runway);
     setArrivalLocation({ lat, lon });
   }
+  function selectAircraft(airplane) {
+    setAircraft(airplane);
+  }
```
Comment (F-Kirchhoff) [inline src/components/Main.js]: do you really need a function for that?

File: src/components/Main.js [hunk 2]
```diff
@@ -110,31 +114,10 @@ export default function Main({ onHandleSubmit }) {
...
-          </option>
-          <option value="B77L" aria-label="boing 777">
-            B777-200LR
-          </option>
-          <option value="SF50" aria-label="vision jet">
-            VISION JET
-          </option>
-        </select>
-      </Aircraft>
+        <Aircraft selectedAircraft={selectAircraft} />
```
Comment (F-Kirchhoff) [inline src/components/Main.js]: selectedAircraft does not sound like a function, but it is! Please rename it to something like onSelectAircraft

File: src/components/features/Aircraft.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import aircraftData from "../initialData/aircraftData";
+
+function Aircraft({ selectedAircraft }) {
+  function onSelect(event) {
+    selectedAircraft(event.target.value);
+  }
```
Comment (F-Kirchhoff) [inline src/components/features/Aircraft.js]: I think you can remove this function as well and call the function from the props directly where you use onSelect

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Luiz, good stuff there, but maybe a bit too overgenineered. You can get rid of two functions since they don't reduce the complexity of your code. Also please rename one of your props such that it is clear that this prop is supposed to be a function.

File: src/components/Main.js (hunk 1 above)
Comment (kornrumpfl) [reply src/components/Main.js]: Thanks for the feedback Felix, all done!

File: src/components/Main.js (hunk 2 above)
Comment (kornrumpfl) [reply src/components/Main.js]: Thanks for the feedback Felix, all done!

File: src/components/features/Aircraft.js (hunk 3 above)
Comment (kornrumpfl) [reply src/components/features/Aircraft.js]: Thanks for the feedback Felix, all done!

File: src/components/features/Aircraft.js [hunk 4]
```diff
@@ -1,13 +1,9 @@
...
 
-function Aircraft({ selectedAircraft }) {
-  function onSelect(event) {
-    selectedAircraft(event.target.value);
-  }
-
+function Aircraft({ onSelectAircraft }) {
   return (
-    <select onChange={onSelect}>
+    <select onChange={(e) => onSelectAircraft(e.target.value)}>
```
Comment (F-Kirchhoff) [inline src/components/features/Aircraft.js]: please do not use e, this is something way to many developers do and its a bad habit. Its not about writing short code, but explicit code. just use event instead.

File: src/components/features/Aircraft.js [hunk 5]
```diff
@@ -3,7 +3,7 @@ import aircraftData from "../initialData/aircraftData";
 
 function Aircraft({ onSelectAircraft }) {
   return (
-    <select onChange={(e) => onSelectAircraft(e.target.value)}>
+    <select onChange={(plane) => onSelectAircraft(plane.target.value)}>
```
Comment (F-Kirchhoff) [inline src/components/features/Aircraft.js]: use event please, even if it seams nitpicky. We want you to write the best code you can, and in this situation the goal is communicate clearly to other developers what is going on.

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]

File: src/components/features/Aircraft.js (hunk 4 above)
Comment (kornrumpfl) [reply src/components/features/Aircraft.js]: Many thanks for the hints, Changes made.

File: src/components/features/Aircraft.js (hunk 5 above)
Comment (kornrumpfl) [reply src/components/features/Aircraft.js]: Many thanks for the hints, Changes made.

**Review by F-Kirchhoff — APPROVED** [review]
Thank you, this LGTM 🦊

**Pull request merged**